roles/tackle/defaults/main.yml

```diff
@@ -24,17 +24,17 @@ hub_deployment_name: "{{ hub_service_name }}"
 hub_deployment_replicas: "1"
 hub_deployment_strategy: "Recreate"
 hub_container_name: "{{ hub_service_name }}"
-hub_container_limits_cpu: "500m"
-hub_container_limits_memory: "800Mi"
-hub_container_requests_cpu: "100m"
-hub_container_requests_memory: "350Mi"
+hub_container_limits_cpu: "1"
+hub_container_limits_memory: "2Gi"
+hub_container_requests_cpu: "500m"
+hub_container_requests_memory: "1Gi"
 hub_database_filename: "hub.db"
 hub_database_volume_name: "{{ hub_service_name }}-database"
-hub_database_volume_size: "5Gi"
+hub_database_volume_size: "10Gi"
 hub_database_volume_path: "/database"
 hub_database_volume_claim_name: "{{ hub_service_name }}-database-volume-claim"
 hub_bucket_volume_name: "{{ hub_service_name }}-bucket"
-hub_bucket_volume_size: "100Gi"
+hub_bucket_volume_size: "25Gi"
```

Comment:
Recent conversations re: static analysis reports may still require upload of large files stored in buckets.  Not totally convinced we know enough to reduce this.